content/en/tracing/trace_collection/automatic_instrumentation/dd_libraries/go.md

```diff
@@ -147,52 +150,92 @@ handler := func(w http.ResponseWriter, r *http.Request) {
-|-------------------------------------|-------------------------------|
-| `database/sql`                      | `v0.7.0`                      |
-| `github.com/gin-gonic/gin`          | `v0.7.0`                      |
-| `github.com/go-chi/chi/v5`          | `v0.7.0`                      |
-| `github.com/go-chi/chi`             | `v0.7.0`                      |
-| `github.com/go-redis/redis/v7`      | `v0.7.0`                      |
-| `github.com/go-redis/redis/v8`      | `v0.7.0`                      |
-| `github.com/gofiber/fiber/v2`       | `v0.7.0`                      |
-| `github.com/gomodule/redigo/redis`  | `v0.7.0`                      |
-| `github.com/gorilla/mux`            | `v0.7.0`                      |
-| `github.com/jinzhu/gorm`            | `v0.7.0`                      |
-| `github.com/labstack/echo/v4`       | `v0.7.0`                      |
-| `google.golang.org/grpc`            | `v0.7.0`                      |
-| `gorm.io/gorm`                      | `v0.7.0`                      |
-| `net/http`                          | `v0.7.0`                      |
-| `go.mongodb.org/mongo-driver/mongo` | `v0.7.3`                      |
-| `k8s.io/client-go`                  | `v0.7.4`                      |
-| `github.com/hashicorp/vault`        | `v0.7.4`                      |
-
-{{% /collapse-content %}}
+To enable the profiler, set the environment variable `DD_PROFILING_ENABLED=true` at runtime.
 
-## Further reading
+### Troubleshooting
 
-{{< partial name="whats-next/whats-next.html" >}}
+To troubleshoot `orchestrion`-managed builds, see [Troubleshooting Go Compile-Time Instrumentation][13]
 
-[1]: /tracing/compatibility_requirements/go
-[2]: https://app.datadoghq.com/apm/service-setup
-[3]: /tracing/trace_collection/library_config/go/
 [4]: https://pkg.go.dev/gopkg.in/DataDog/dd-trace-go.v1/ddtrace
-[5]: /tracing/trace_collection/automatic_instrumentation/?tab=datadoglibraries#install-and-configure-the-agent
 [6]: https://github.com/DataDog/orchestrion
 [7]: /security/application_security/threats/exploit-prevention
 [8]: https://go.dev/doc/devel/release#policy
-[9]: https://www.datadoghq.com/support/
 [10]: https://pkg.go.dev/cmd/go#hdr-Modules__module_versions__and_more
 [11]: https://github.com/DataDog/orchestrion/releases
-[12]: /profiler
\ No newline at end of file
+[12]: /profiler
+[13]: /tracing/troubleshooting/go_compile_time/
+[14]: /getting_started/tagging/unified_service_tagging/
+[15]: /security/application_security/threats/exploit-prevention/
+
+{{% /tab %}}
+
+{{% tab "Manual instrumentation" %}}
+
+### Activate Go integrations to create spans
+
+Datadog has a series of pluggable packages which provide out-of-the-box support for instrumenting a series of libraries and frameworks. A list of these packages can be found in the [Compatibility Requirements][1] page. Import these packages into your application and follow the configuration instructions listed alongside each [Integration][1].
```

Comment:
Is `[Integration][1]` a mistake here?

Reply:
I guess the "pluggable packages" are also called "integrations" (or "contribs", although that's not a name I'd use in the docs), and they are still listed at `[1]`.

Reply:
I don't think it makes sense to link to the same exact link twice in the same paragraph. Also, assuming this "integration" is different from a Datadog integration (which it seems to be), I would like to use a different word. If it's "pluggable packages", so be it.

Reply:
Changing to `package` (avoiding to repeat "pluggable" because what does it even mean?)